src/Skeleton/BaseController.php

```diff
@@ -274,7 +299,7 @@ protected function setStatusCode($statusCode)
      */
     protected function respondWithItem($item)
     {
-        $resource = new Item($item, $this->transformer);
+        $resource = new Item($item, $this->transformer, $this->resourceKey);
```

Comment:
I've also would have gone another way - `$resourceKey='car'` and use `str_plural`